PRIVACY.md

```diff
@@ -35,11 +38,17 @@ monitored on a regular basis and updated according to technical progress as
 necessary. Our staff is well informed about privacy rules and obliged to follow
 them.
 
-2.5. As provided for by applicable provisions we are entitled to disclose and
-submit data for criminal prosecution and risk prevention purposes.
+2.5. Furthermore, we will not pass on personal data without your express
```

Comment:
Clarify consent with regard to passing on data, and keep GDPR granted exceptions for criminal prosecution, regulations and son on.